docker/cloudberry_tests/scripts/tests/restore_point.sh

```diff
@@ -18,6 +18,38 @@ wal-g --config=${TMP_CONFIG} delete everything FORCE --confirm
 wal-g create-restore-point rp1 --config=${TMP_CONFIG}
 wal-g create-restore-point rp2 --config=${TMP_CONFIG}
 
+# Check whether the WAL log is correctly switched and uploaded to S3 after create-restore-point
+# gpadmin@10f4a227f02b:/usr/local/gpdb_src$ wal-g st ls  segments_005/seg0/wal_005/ --config=${TMP_CONFIG}
+# type size    last modified                     name
+# obj  4624920 2025-05-21 07:17:16.052 +0000 UTC 000000010000000000000001.lz4
+# obj  264275  2025-05-21 07:26:06.265 +0000 UTC 000000010000000000000002.lz4
+
+#wait for wal-g to upload WALs
+sleep 5
+
+check_wal_upload() {
+    local path=$1
+
+    wal-g st ls "$path" --config=${TMP_CONFIG}
+
+    wal-g st ls "$path" --config=${TMP_CONFIG} \
+        | awk '/^obj/ {count++} END {exit !(count >= 2)}'
+}
```

Comment:
Probably, wal-g on its own should wait for WAL upload before signaling that backup is ready.
But it is out of scope for this PR.

cc: @debebantur